src/sentry/scripts/utils/locking/delete_lock.lua

```diff
@@ -0,0 +1,12 @@
+local key = KEYS[1]
+local uuid = ARGV[1]
+
+local value = redis.call('GET', key)
+if not value then
+    return redis.error_reply(string.format("No lock at key exists at key: %s", key))
+elseif value ~= uuid then
+    return redis.error_reply(string.format("Lock at %s was set by %s, and cannot be released by %s.", key, value, uuid))
+else
+    redis.call('DEL', key)
```

Comment:
Ignore me, I guess it'll never error, it'll just return `0` if no keys were deleted.

Alternatively, for the `status_reply`, could/should we just `return redis.status_reply('OK')`? It seems to fit better with other redis responses, but not sure how this manifests to the client and whatnot.